python/ray/rllib/policy_gradient/policy_gradient.py

```diff
@@ -22,6 +23,8 @@
     "gamma": 0.995,
     # Number of steps after which the rollout gets cut
     "horizon": 2000,
+    # If true, use the GAE estimator (with a value function)
```

Comment:
Can you rewrite this as 

```
# If true, use the Generalized Advantage Estimator (GAE) with a value function.
```

Otherwise people probably may not know the acronym.